dbsnp-importer/src/test/resources/schema.sql

```diff
@@ -0,0 +1,2802 @@
+
+--
+-- Name: subpopallele_type_idx; Type: INDEX; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+CREATE INDEX subpopallele_type_idx ON subpopallele (type);
+
+
+--
+-- Name: subpopgty_gty_id_idx; Type: INDEX; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+CREATE INDEX subpopgty_gty_id_idx ON subpopgty (gty_id);
+
+
+--
+-- Name: subpopgty_subpop_id_idx; Type: INDEX; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+CREATE INDEX subpopgty_subpop_id_idx ON subpopgty (subpop_id);
+
+
+--
+-- Name: subsnp_batch_id_subsnp_id_idx; Type: INDEX; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+CREATE INDEX subsnp_batch_id_subsnp_id_idx ON subsnp (batch_id, subsnp_id);
+
+
+--
+-- Name: subsnp_loc_snp_id_upp_subsnp_id_idx; Type: INDEX; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+CREATE INDEX subsnp_loc_snp_id_upp_subsnp_id_idx ON subsnp (loc_snp_id_upp, subsnp_id);
+
+
+--
+-- Name: subsnp_subsnp_id_batch_id_loc_snp_id_idx; Type: INDEX; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+CREATE INDEX subsnp_subsnp_id_batch_id_loc_snp_id_idx ON subsnp (subsnp_id, batch_id, loc_snp_id);
+
+
+--
+-- Name: subsnpacc_acc_part_acc_type_ind_subsnp_id_idx; Type: INDEX; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+CREATE INDEX subsnpacc_acc_part_acc_type_ind_subsnp_id_idx ON subsnpacc (acc_part, acc_type_ind, subsnp_id);
+
+
+--
+-- Name: subsnpmdfailln_subsnp_id_idx; Type: INDEX; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+CREATE INDEX subsnpmdfailln_subsnp_id_idx ON subsnpmdfailln (subsnp_id);
+
+
+--
+-- Name: subsnppubmed_pubmed_id_idx; Type: INDEX; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+CREATE INDEX subsnppubmed_pubmed_id_idx ON subsnppubmed (pubmed_id);
+
+
+--
+-- Name: fk_b148_snpcontigloc_rs; Type: FK CONSTRAINT; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+ALTER TABLE b148_snpcontigloc
+    ADD CONSTRAINT fk_b148_snpcontigloc_rs FOREIGN KEY (snp_id) REFERENCES snp(snp_id);
+
+
+--
+-- Name: fk_b148_snpmapinfo_rs; Type: FK CONSTRAINT; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+ALTER TABLE b148_snpmapinfo
+    ADD CONSTRAINT fk_b148_snpmapinfo_rs FOREIGN KEY (snp_id) REFERENCES snp(snp_id);
+
+
+--
+-- Name: fk_subpop_batch_id; Type: FK CONSTRAINT; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+ALTER TABLE subpop
+    ADD CONSTRAINT fk_subpop_batch_id FOREIGN KEY (batch_id) REFERENCES batch(batch_id);
+
+
+--
+-- Name: fk_subpop_pop; Type: FK CONSTRAINT; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+ALTER TABLE subpop
+    ADD CONSTRAINT fk_subpop_pop FOREIGN KEY (pop_id) REFERENCES population(pop_id);
+
+
+--
+-- Name: fk_subpopallele_bid_ss_pop_type; Type: FK CONSTRAINT; Schema: dbsnp_cow; Owner: dbsnp
+--
+
+ALTER TABLE subpopallele
+    ADD CONSTRAINT fk_subpopallele_bid_ss_pop_type FOREIGN KEY (batch_id, subsnp_id, pop_id, type) REFERENCES subpop(batch_id, subsnp_id, pop_id, type);
+
+
+--
+-- Name: dbsnp_cow; Type: ACL; Schema: -; Owner: dbsnp
+--
+
+--REVOKE ALL ON SCHEMA dbsnp_cow FROM PUBLIC;
+--REVOKE ALL ON SCHEMA dbsnp_cow FROM dbsnp;
+--GRANT ALL ON SCHEMA dbsnp_cow TO dbsnp;
+--GRANT USAGE ON SCHEMA dbsnp_cow TO dbsnp_ro;
+
+
+--
```

Comment:
I'm not sure it is worth keeping this huge commented block in the script. It is version-specific so we can't even use it as backup for the latest release.